be/src/storage/lake/cloud_native_persistent_index.cpp

```diff
@@ -0,0 +1,76 @@
+// Copyright 2021-present StarRocks, Inc. All rights reserved.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+#include "storage/lake/cloud_native_persistent_index.h"
+
+namespace starrocks::lake {
+
+CloudNativePersistentIndex::CloudNativePersistentIndex(std::string path) : PersistentIndex(std::move(path)) {
```

Comment:
The class is already in the namespace `lake`, is it still necessary to add the `CloudNative` prefix?

Reply:
But it seems that little wired that `PersistentIndex : PersistentIndex` ?

Reply:
If so, how about `LakePersistentIndex`, following the current naming convention.